include/rest/models/modify_positions_args.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef _modify_positions_args_hpp_
+#define _modify_positions_args_hpp_
+
+#include <models/model.hpp>
+
+namespace disccord
+{
+    namespace rest
+    {
+        namespace models
+        {
+            class modify_positions_args : public disccord::models::model
+            {
+                public:
+                    modify_positions_args(std::vector<std::pair<uint64_t, uint8_t>> id_pos_params);
+                    virtual ~modify_positions_args();
+
+                    std::vector<std::pair<uint64_t, uint8_t>> get_id_pos_params();
+
+                protected:
+                    virtual void encode_to(std::vector<web::json::value>& info) override;
```

Comment:
Now that I look at this, this shouldn't work: in `disccord::models::model` we explicitly encode to `web::json::value::object`, which expects a `std::vector<std::pair<std::string, web::json::value>>`